content/concept-amendments.md

```diff
@@ -121,6 +122,16 @@ A transaction remains in the queue until one of the following happens:
 * It becomes invalid (for example, the [`LastLedgerSequence`](reference-transaction-format.html#lastledgersequence) causes it to expire)
 * It gets dropped because there are too many transactions in the queue with a higher transaction cost.
 
+## FlowV2 ##
+
+| Amendment ID | Status |
+|--------------|--------|
+| 5CC22CFF2864B020BD79E0E1F048F63EF3594F95E650E43B3F837EF1DF5F4B26 | Planned for voting, expected to be enabled 2016-08-24 |
+
+Replaces the payment processing engine with a complete rewrite. The new version of the payment processing engine is intended to follow the same rules as the old one, but occasionally produces different results due to floating point rounding.
```

Comment:
The floating point rounding occurring here is internal math (mostly related to calculating order quality, I think). It's not directly related to the precision represented in that link, so linking there would probably be more misleading than helpful.